twisted/plugins/tunnel_helper_plugin.py

```diff
@@ -23,10 +24,48 @@
 from Tribler.Core.simpledefs import NTFY_TUNNEL, NTFY_REMOVE
 from Tribler.dispersy.tool.clean_observers import clean_twisted_observers
 
-
 # Register yappi profiler
 from Tribler.dispersy.utils import twistd_yappi
 
+def is_valid_ipv4(ip):
```

Comment:
You can steal a shorter version from Dispersy:

https://github.com/Tribler/dispersy/blob/7a2893bb4110da79ec7300652aa7baa4afb40a91/util.py#L247

Reply:
I don't want to add another dependency on Dispersy. And I am not even sure that this check should be there. We got no policy on this kind of checks.

Reply:
Ok, changed that to use "inet_aton".